cms/models.py

```diff
@@ -59,4 +60,60 @@ class ProgramPage(Page):
     content_panels = Page.content_panels + [
         FieldPanel('description', classname="full"),
         FieldPanel('program'),
+        InlinePanel('FAQs', label='Frequently Asked Questions'),
+        InlinePanel('Courses', label='Program Courses')
+    ]
+
+    def get_context(self, request):
+        js_settings = {
+            "gaTrackingID": settings.GA_TRACKING_ID,
+            "host": webpack_dev_server_host(request)
+        }
+
+        context = super(ProgramPage, self).get_context(request)
+
+        context["style_src"] = get_bundle_url(request, "style.js")
+        context["public_src"] = get_bundle_url(request, "public.js")
+        context["style_public_src"] = get_bundle_url(request, "style_public.js")
+        context["authenticated"] = not request.user.is_anonymous()
+        context["username"] = request.user.username
+        context["js_settings_json"] = json.dumps(js_settings)
+        context["title"] = self.title
+
+        return context
+
+
+class Courses(Orderable):
+    """
+    Courses listed for the program
+    """
+    program_page = ParentalKey(ProgramPage, related_name='Courses')
+    title = models.CharField(max_length=255, default='')
+    description = RichTextField(blank=True, null=True)
+    content_panels = Page.content_panels + [
+        MultiFieldPanel(
+            [
+                FieldPanel('title'),
+            ]
+        )
+    ]
+
+
+class FrequentlyAskedQuestions(Orderable):
+    """
+    FAQs for the program
+    """
+    page = ParentalKey(ProgramPage, related_name='FAQs')
```

Comment:
Can you rename to `program_page` for consistency with its use in the above model?